GOVERNANCE.md

```diff
@@ -0,0 +1,195 @@
+# Governance
+
+This document outlines the governance model for Biome. This document includes the contributor model, code review, merging, and the consequences and process for Code of Conduct violations.
+
+## Member Roles
```

Comment:
Are there specific criteria for someone to be considered a "Core Contributor" or "Maintainer"?
Are there any conditions under which a Lead might lose their status or privileges?

Reply:
> Are there specific criteria for someone to be considered a "Core Contributor" or "Maintainer"? 

A core maintainer should be first a maintainer. Other than that, I don't think there are specific criteria. Do you have anything in mind?

> Are there any conditions under which a Lead might lose their status or privileges?

I would say: inactivity, violation of CoC and step-down. Should we write it somewhere? If so, where?